PowerShellTools/PowerShellToolsPackage.cs

```diff
@@ -125,11 +124,11 @@ public PowerShellToolsPackage()
         /// </summary>
         public static PowerShellToolsPackage Instance { get; private set; }
 
-        public static IPowershellIntelliSenseService IntelliSenseService
+        internal static IPowershellIntelliSenseService IntelliSenseService
         {
             get
             {
-                return ConnectionManager.PowershellIntelliSenseSerivce;
+                return _connectionManager.PowershellIntelliSenseSerivce;

```

Comment:
Consider to check null to prevent null reference if consumer try to access it without establishconnection.